src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
```

Comment:
To address the comment below, that function should accept an array of `cwds` instead of a single `cwd` and should look for plugin in the order paths are in the array.

Then, we can just change:
https://github.com/rnpm/rnpm/blob/bug/using-rnpm-locally/src/getCommands.js#L29
from
`findPlugins(rnpmRoot).map(getPluginCommands(rnpmRoot))`
to
`findPlugins(rnpmRoot).map(getPluginCommands([rnpmRoot, appRoot]))`
and problem solved.